tests/loss/test_koleo_loss.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import Tensor
+
+from lightly.loss.koleo_loss import KoLeoLoss
+
+
+class TestKoLeoLoss:
+    # Test values generated using the original implementation from:
+    # https://github.com/facebookresearch/dinov2/blob/main/dinov2/loss/koleo_loss.py
+    @pytest.mark.parametrize(
+        "x, expected_loss",
+        [
+            (torch.tensor([[1.0]]), 17.7275),
+            (torch.tensor([[1.0, 1.0]]), 17.5393),
```

Comment:
These two don't make sense, as they only calculate `-ln(eps + float_inaccuracies)`. I would only keep one of them, to ensure that a batch size of 1 does not break.

However, can you add these?
- [[0.0], [1.0]] -> should give mean([-ln(1), -ln(1)])
- [[0.0], [1.0], [2.0] -> should give mean([-ln(1), -ln(2)])

Reply:
Added more tests :)

Note that tests like this don't really make sense: `[[0.0], [1.0], [2.0]]`. The reason is that the loss first normalizes the vectors (`[0.0]`, `[1.0]`, `[2.0]`) to unit norm. So you'll always end up with a `[1.0]` vector (except for 0.0 which remains 0.0).